### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by BharathKmalviya**
## :camera: Screenshots

`````
val client = OkHttpClient.Builder()
                .addInterceptor(ChuckerInterceptor.Builder(context).build())
                .build()
`````

## :pencil: Changes

- Updated the latest way in example to add Interceptor in OkHttpClient.Builder()

### CODE REVIEW

**Comment by yogeshpaliyal** [comment]
@BharathKmalviya 
This should be 
```kotlin
val client = OkHttpClient.Builder()
                .addInterceptor(ChuckerInterceptor.Builder(context).build())
                .build()
```

**Comment by BharathKmalviya** [comment]
@yogeshpaliyal 
oh yeah, Thanks!!

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ To start using Chucker, just plug it a new `ChuckerInterceptor` to your OkHttp C
 
 ```kotlin
 val client = OkHttpClient.Builder()
-                .addInterceptor(ChuckerInterceptor(context))
+                .addInterceptor(ChuckerInterceptor.Builder(context))
```
Comment (yogeshpaliyal) [inline README.md:49]:
```suggestion
.addInterceptor(ChuckerInterceptor.Builder(context).build())
```

**Comment by MiSikora** [comment]
Closing for the same reason mentioned in #706.

**Pull request closed without merging**